.circleci/config.yml

```diff
@@ -192,13 +192,101 @@ jobs:
+            # Elasticsearch requires to increase this setting's default value
+            sudo sysctl -w vm/max_map_count=262144
+            sudo bash -c "echo '{\"insecure-registries\": [\"172.30.0.0/16\"]}' > /etc/docker/daemon.json"
+            sudo service docker restart
+
+      - run:
+          name: Run local OpenShift cluster & configure environment
+          command: |
+            K8S_DOMAIN=$(hostname -I | awk '{print $1}')
+            export K8S_DOMAIN
+            oc cluster up --server-loglevel=5 --public-hostname=${K8S_DOMAIN}
+            oc login https://${K8S_DOMAIN}:8443 -u developer -p developer --insecure-skip-tls-verify=true
+            # Set environment variables for the CI
+            echo "export K8S_DOMAIN=${K8S_DOMAIN}" >> $BASH_ENV
+            echo "export ARNOLD_DEFAULT_VAULT_PASSWORD=arnold" >> $BASH_ENV
+            echo 'export ANSIBLE_VAULT_PASSWORD="${ARNOLD_DEFAULT_VAULT_PASSWORD}"' >> $BASH_ENV
+            echo "export K8S_AUTH_VERIFY_SSL=no" >> $BASH_ENV
+            source $BASH_ENV
+
+      - run:
+          name: Install arnold CLI
+          command: |
+            wget https://raw.githubusercontent.com/openfun/arnold/master/bin/arnold -O ${HOME}/bin/arnold
+            chmod +x $HOME/bin/arnold
+
+      - run:
+          name: Setup a new Arnold project
+          command: |
+            arnold -c ralph -e ci setup
+            arnold -d -c ralph -e ci -a ralph create_app_vaults
+            arnold -d -c ralph -e ci -- vault -a ralph decrypt
+            sed -i 's/^# RALPH_/RALPH_/g' group_vars/customer/ralph/ci/secrets/ralph.vault.yml
+            arnold -d -c ralph -e ci -- vault -a ralph encrypt
+            VARS_FILE="group_vars/customer/ralph/ci/main.yml"
+            echo -e "skip_verification: True" > ${VARS_FILE}
+            echo -e "apps:\n  - name: ralph" >> ${VARS_FILE}
+            echo -e "ralph_image_name: 172.30.1.1:5000/ci-ralph/ralph" >> ${VARS_FILE}
+            echo -e "ralph_image_tag: ${CIRCLE_SHA1}" >> ${VARS_FILE}
+            echo -e "ralph_app_replicas: 1" >> ${VARS_FILE}
+            echo -e "ralph_cronjobs:" >> ${VARS_FILE}
+            echo -e "  - name: ci-test" >> ${VARS_FILE}
+            echo -e "    schedule: '* * * * *'" >> ${VARS_FILE}
+            echo -e "    command: ['date']" >> ${VARS_FILE}
+
+      - run:
+          name: Push ralph image to the k8s cluster docker registry
+          command: |
+            docker login -u developer -p "$(oc whoami -t)" 172.30.1.1:5000
+            docker tag ralph:${CIRCLE_SHA1} 172.30.1.1:5000/ci-ralph/ralph:${CIRCLE_SHA1}
+            arnold -d -c ralph -e ci create_project
+            docker push 172.30.1.1:5000/ci-ralph/ralph:${CIRCLE_SHA1}
+
+      - run:
+          name: Bootstrap ralph application
+          command: |
+            arnold -d -c ralph -e ci -a ralph init
+            arnold -d -c ralph -e ci -a ralph deploy
+            oc project ci-ralph
+            oc get pods -l app=ralph | grep Running
```

Comment:
Maybe we can also check that at least one cronjob ran?

Reply:
In this case we should increase the job frequency to at least 10 sec.

Reply:
If you want to check this, I recommend to look [here](https://www.craftypenguins.net/how-to-trigger-a-kubernetes-cronjob-manually/) and [here](https://www.mankier.com/1/oc-create-job). You can manually trigger a cronjob execution, instead of waiting for it to be scheduled.

Reply:
I'll follow @madmatah instructions on this.